lib/workers/repository/init/index.ts

```diff
@@ -37,7 +66,13 @@ export async function initRepo(
   let config: RenovateConfig = initializeConfig(config_);
   await initializeCaches(config);
   config = await initApis(config);
-  config = await getRepoConfig(config);
+  // istanbul ignore if
```

Comment:
We should  better add a simple test for this. run this function twice should do it i think.